stuff/doc/LICENSE/LICENSE_win8.txt

```diff
@@ -0,0 +1,23 @@
+Windows Pointer Input Message support code files from Krita
```

Comment:
It may be a a matter of preference, but can you change the name of the license file from `LICENSE_win8.txt` to something more illustrative one?
Well, let's say, how about `LICENSE_kistabletsupportwin8.txt` or `LICENSE_winink.txt`?